core/src/main/java/org/polypheny/db/catalog/snapshot/Snapshot.java

```diff
@@ -41,33 +40,32 @@
 
 public interface Snapshot extends OperatorTable {
 
-    NameMatcher nameMatcher = NameMatchers.withCaseSensitive( RuntimeConfig.RELATIONAL_CASE_SENSITIVE.getBoolean() );
+    NameMatcher nameMatcher = NameMatchers.withCaseSensitive( false );
```

Comment:
Why change to false? Shouldn't it get the case-sensitive setting from the parent namespace?

Reply:
This was mostly still used as an initial value for matching various objects. Moved it to these places and removed it from snapshot.